### PR TITLE
vendor in new opencontainers/runtime-tools

### PR DESCRIPTION
**Pull request by baude**
mrunal has asked me to split the vendoring of this package into its
own PR and separate from the libpod integration work.

additional changes to reflect an API change with generate.New() which
now requires a string representation of the operating system and now
also returns an error.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: vendor.conf [hunk 1]
```diff
@@ -21,7 +21,7 @@ google.golang.org/grpc 5b3c4e850e90a4cf6a20ebd46c8b32a0a3afcb9e https://github.c
 google.golang.org/genproto 09f6ed296fc66555a25fe4ce95173148778dfa85
 github.com/opencontainers/selinux b6fa367ed7f534f9ba25391cc2d467085dbb445a
 github.com/opencontainers/go-digest v1.0.0-rc0
-github.com/opencontainers/runtime-tools fdbc3d6d9507f699bbfd557dce0640c02b5f60e4
+github.com/opencontainers/runtime-tools 05b5772e41b86f67d13df2d039eb839923a5e93e
```
Comment (wking) [inline vendor.conf]: This works, but I'd rather use the v0.6.0 tag or its associated commit.

Comment (baude) [reply vendor.conf]: fixed ....

**Pull request merged**